proxmoxtf/resource/file.go

```diff
@@ -95,6 +96,15 @@ func File() *schema.Resource {
 				Description: "The file name",
 				Computed:    true,
 			},
+			mkResourceVirtualEnvironmentFileFileMode: {
+				Type: schema.TypeString,
+				Description: `The file mode in octal format, e.g. "0700" or "600".` +
```

Comment:
The new attribute should also be documented in https://github.com/bpg/terraform-provider-proxmox/blob/2e34c57f6c95bff2f3e35ee617e783a78616c900/docs/resources/virtual_environment_file.md?plain=1#L99

Would you mind adding a note about the required `root@pam` auth as well, as per the comment above? 

Unfortunately, it is still a manual step for old SDKv2 resources. New resources implemented with the Plugin Framework have their docs auto-generated.

Reply:
Sure, I extended the description for the `hook_script_file_id` attribute but forgot to add the docs for the new attribute at all 🙈